test/k8sT/manifests/cnp-ingress-from-cidr-to-ports.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+description: "Policy to allow traffic external to cluster inwards via CIDR and port"
+metadata:
+  name: "cnp-ingress-from-cidr-to-ports"
+spec:
+  endpointSelector:
+    matchLabels:
+      zgroup: testDS
+  ingress:
+  - toPorts:
+    - ports:
+      - port: "80"
+        protocol: TCP
+    fromCIDR:
+      - 192.168.36.0/24
+      - 192.168.36.13/32
```

Comment:
Why are both CIDRs specified here and not only the first?

Reply:
Ah, good catch. I only meant for `192.168.36.13/32` to be in there.